app/src/main/java/io/github/droidkaigi/confsched/api/DroidKaigiClient.java

```diff
@@ -36,14 +37,26 @@ public DroidKaigiClient(OkHttpClient client) {
         service = feedburnerRetrofit.create(DroidKaigiService.class);
     }
 
-    public Observable<List<Session>> getSessions() {
-        return service.getSessions();
+    public Observable<List<Session>> getSessions(String languageId) {
+        if ("ja".equals(languageId)) {
+            return service.getSessionsJa();
+        } else if ("ar".equals(languageId)) {
+            return service.getSessionsAr();
+        } else {
+            return service.getSessionsEn();
+        }
     }
 
     public interface DroidKaigiService {
 
-        @GET("/konifar/droidkaigi2016/master/app/src/main/res/raw/sessions_ja.json")
-        Observable<List<Session>> getSessions();
+        @GET(SESSIONS_API_ROUTES + "sessions_ja.json")
+        Observable<List<Session>> getSessionsJa();
+
+        @GET(SESSIONS_API_ROUTES + "sessions_en.json")
+        Observable<List<Session>> getSessionsEn();
+
+        @GET(SESSIONS_API_ROUTES + "sessions_ar.json")
+        Observable<List<Session>> getSessionsAr();
```

Comment:
Load api for multi languages.
@hkurokawa Please put `sessions_en.json` in other PR :)